rpc/client/examples_test.go

```diff
@@ -0,0 +1,126 @@
+package client_test
+
+import (
+	"bytes"
+	"fmt"
+
+	"github.com/tendermint/tendermint/abci/example/kvstore"
+	"github.com/tendermint/tendermint/rpc/client"
+	ctypes "github.com/tendermint/tendermint/rpc/core/types"
+	rpctest "github.com/tendermint/tendermint/rpc/test"
+)
+
+func ExampleHTTP_simple() {
```

Comment:
This is not according to Go naming. Why not `ExampleHTTPSimple`?

Reply:
See under "Example function names":
https://blog.golang.org/examples

Reply:
ooo, okay 👍 I'll read

Reply:
See [here](https://golang.org/pkg/testing/#hdr-Examples) - since I'm not documenting a specific function, using a `_suffix` makes more sense, since these are package-level examples that are conceptually differentiated, instead of being specific to a particular function/type/method.